src/Components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  tabId?: string;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  tabId,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === tabId) || tabs[0];
+
```

Comment:
```suggestion
}) => {
  const selectedTab = tabs.find(tab => tab.id === tabId);

```
That's why you got 'Some text 1' content when no tab was selected